src/app.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const readline = require('readline');
+const { digitGenerator } = require('./digitGenerator.js');
+const { compareDigit } = require('./compareDigit.js');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let compareResult = {
+  bull: 0,
+  cow: 0,
+};
+let amountOfDigits;
+let randomDigits;
+
+const getAmount = () => {
+  rl.question('Choose amount of digits you want to guess, from 1 to 9: ',
```

Comment:
why from 1 to 9?  `0` also digit and in guess number was `0`
![Screenshot from 2023-07-25 15-58-54](https://github.com/mate-academy/node_bulls-and-cows/assets/33029864/1d5c51b7-f9c5-4e25-83f5-eed5c9ab9e0e)